pkgs/mempool/generate.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p gnupg gnused jq
+set -euo pipefail
```

Comment:
Is it expected that the hashes in "frontend-assets-fetch.sh" and `frontendAssets` in "mempool/default.sh" change when running "generate.sh"? It seems like for the frontend assets we look for there most recent ones regardless of the mempool version we try to "generate".

Reply:
The frontend assets are not linked to a specific mempool version and are updated continuously.
I've removed the liquid assets. These are unused (we don't support liquid) and are updated daily.
Now we're left with mining pool assets, which only change every few months.